### ISSUE TITLE
Pytorch Lightning deprecated "gpus" argument in trainer

### ISSUE THREAD

**Issue by alicerdv** [RelationRx/pyrelational#14]
Pytorch lightning will remove the "gpus" argument in the trainer in favour of using "accelerator" and "devices". See docs:
https://pytorch-lightning.readthedocs.io/en/stable/api/pytorch_lightning.trainer.trainer.Trainer.html?highlight=trainer

**Comment by ochalkley-advai** [comment]
This now causes an error when trying to run the quick start tutorial. How can we fix this?

`TypeError: __init__() got an unexpected keyword argument 'gpus'`

**Comment by thomasgaudelet** [comment]
Thanks @ochalkley-advai for raising this. We're working to fix it now!

**Comment by ochalkley-advai** [comment]
Thanks @thomasgaudelet, I tried PyTorch 1.13 but that didn't work strangely. I did find a hacky workaround of commenting out the gpus arg in the `pl.Trainer` call on line 83 of `pyrelational/model_managers/lightning_model_manager.py` and adding `accelerator="auto",`. This should do for the time being but I would definitely like a proper fix ASAP. I don't have experience contributing to open-source projects but happy to help out on this if there is anything I can do?
